templates/company-board/README.md

```diff
@@ -71,3 +73,16 @@ The network details will be automatically selected by the `arapp.json`'s environ
 | Board Token Manager | BURN                  | Share Voting        | Share Voting  |
 | Share Token Manager | MINT                  | Share Voting        | Share Voting  |
 | Share Token Manager | BURN                  | Share Voting        | Share Voting  |
```

Comment:
For all the templates, we should also clarify the Vault and Agent's permissions.

In particular, we should use `Vault / Agent` when the permissions are always created, and use another heading for the optional permissions created just when the Agent is installed.